app/views/users/_user_list.html.erb

```diff
@@ -14,7 +14,7 @@
                       <p><%= presenter.title_list %></p>
                   <% end %>
                   <% if user.country_name %>
-                      <p><%= presenter.country_name %></p>
+                      <p title="<%= presenter.country_name unless presenter.country_name.length < 15 %>"><%= truncate(presenter.country_name, length:14) %></p>
```

Comment:
We could - I was worried about making that page bigger and therefore slower than it already is.
hmmm...on mobile it's even more of an issue - you just won't want the tooltips at all because most mobiles won't support the idea of "hover over" in order to see the tooltip....though mobile is a separate PR :}
So how about I always add the tooltip on md and lg screen sizes but never add it on sm and xs?